dev-tools/README.md

```diff
@@ -1,27 +1,41 @@
-## Setup Instructions
+# Setup Instructions
 
 ```
 # initial setup
 curl "https://raw.githubusercontent.com/AudiusProject/audius-protocol/main/dev-tools/setup.sh" | bash
 
 # refresh terminal for docker
 exit
-
-# build and pull
+```
+```
+# build and run
 audius-compose build # About 10 minutes
```

Comment:
can this just be 

```
audius-compose up
```

doesn't that implicitly build?

Reply:
separating these steps is good

Reply:
Since the first build, builds the cache

Reply:
kk @cheran-senthil